internal/cmd/local/local/cmd.go

```diff
@@ -357,6 +356,7 @@ func (c *Command) Install(ctx context.Context, opts InstallOpts) error {
 		fmt.Sprintf("global.env_vars.AIRBYTE_INSTALLATION_ID=%s", telUser),
 		fmt.Sprintf("global.jobs.resources.limits.cpu=3"),
 		fmt.Sprintf("global.jobs.resources.limits.memory=4Gi"),
+		//fmt.Sprintf("global.auth.enabled=true"),
```

Comment:
@colesnodgrass This should be the same as setting

```
global:
  auth
    enabled: true
```

right?

Reply:
Correct.
---
Unrelated to this question, but in looking at this code, there is to reason to use `fmt.Sprintf` here, or in the two lines above (which I originally wrote 😞)

Reply:
No, I was just following your lead. A string would be fine.